### PR TITLE
Add a little script that prints out invalid users in the db

### PR DESCRIPTION
**Pull request by Smephite**
Depends on #16 

prints all users in `votes_history` not being a Lumenaut and puts them into WHERE clause to use in _live_ production databases ;)

### CODE REVIEW

**Comment by w00kie** [comment]
Sorry, I think that because of our little issue with my work account and the rebasing, it messed up your pull request. You might have to redo it 😓

File: handle_clean_db.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
+from helper import *
+from main import *
+from discord_helpers import hasRole
+
+@client.event
+async def on_ready():
+    print(f'We have logged in as {client.user}')
+    leaderboard = fetchLeaderboard(conn)
+    to_be_removed = []
+    guild = client.get_guild(763798356484161566)
```
Comment (MatejMecka) [inline handle_clean_db.py:10]: Can it be an env Variable?

**Review by MatejMecka — CHANGES_REQUESTED** [review]
Just one little thing

**Review by MatejMecka — APPROVED** [review]
LGTM!

**Pull request merged**